tests/functional/src/Classes/BlockControl.php

```diff
@@ -23,8 +24,12 @@ class BlockControl extends Control
     public function render(): void
     {
         $this->getTemplate()->privilege = $this->privilege;
-        $this->template->setFile(__DIR__.'/../../templates/block.latte');
-        $this->template->render();
+
+        /** @var ITemplate $template */
+        $template = $this->template;
```

Comment:
The `0.9.x-dev` version of PHPStan seems more picky about stuff like this. I think in this case it was complaining that `$this->template` could also be an instance of `stdClass`.